### DISCUSSION TITLE
macOS Menu Bar Icon Transparency?

### DISCUSSION THREAD

**Discussion by jacobjivanov** [rxhanson/Rectangle#304] [Ideas]
![image](https://user-images.githubusercontent.com/59805681/102923909-9279c300-445e-11eb-8550-618952aecd47.png)

Why does the macOS (Big Sur) Menu Bar Icon for Rectangle seem slightly transparent? It's bothering my OCD.

**Comment by rxhanson** [comment]
In v0.41, I made it less transparent.
https://github.com/rxhanson/Rectangle/releases/tag/v0.41

It was 90% opaque, and I upped it to 95% opaque.

It has transparency mainly to make the icon stand out less, since it's just a solid rectangle and it looked worse to me having it so bright. It definitely looks worse on your screenshot than on my machine though, but maybe it's just me. 

If you don't mind, give me another screenshot and I'll see if I want to up it to eye-blistering full opacity.

**Reply by jacobjivanov** [reply]
<img width="870" alt="Screen Shot 2021-01-01 at 11 52 04 PM" src="https://user-images.githubusercontent.com/59805681/103451003-72cb7180-4c8c-11eb-9ad0-8c927a78e81d.png">
(changed wallpaper)

I don't really think 100% opaque would look bad at all. In fact, I think it'd calm my and other's OCD. (even though I can't tell the difference without a color picker, it bothers me). 


On a different note, I downloaded Magnet, because I heard about how apparently amazing they were, and... I was disappointed. The only real thing I can agree is that they have a better icon and logo, and to some extent, I like the UI of the pop-up menu better . For example, here is a comparison of your pop-up menu, and there's:
<img width="302" alt="Screen Shot 2021-01-01 at 11 58 03 PM" src="https://user-images.githubusercontent.com/59805681/103451065-39dfcc80-4c8d-11eb-86c2-13278e0c785f.png">
<img width="287" alt="Screen Shot 2021-01-01 at 11 57 48 PM" src="https://user-images.githubusercontent.com/59805681/103451067-3cdabd00-4c8d-11eb-924f-beafb50c8633.png">

While I do notice the *startling* similarity, I also notice the lack of some options in Magnet, i.e. my favorite one "Almost Maximize", or the never used "Sixths" and rarely used "Fourths". I might have to complain and get a refund. In fact, if you released this on the macOS app store, I think it would quickly overtake Magnet, Tile, and the 100 other versions, because from what I've seen, this one is simply the best.

Idea: Be able to create a saved window dimensions and position, maybe with some kind of helper tool to create them, with options like "window width %", etc.

**Comment by rxhanson** [comment]
> The only real thing I can agree is that they have a better icon and logo

How dare you (haha I kid). I'm no designer, so I will gladly accept a PR with some status bar and window icon updates. The AppIcon looks good to me, so I wouldn't change that. Magnet updated their icons not long ago, maybe for Big Sur.

>  I might have to complain and get a refund. In fact, if you released this on the macOS app store, I think it would quickly overtake Magnet, Tile, and the 100 other versions

Rather than request a refund, request some things from Apple:

1. That they allow new window management apps to be added to the App Store. The existing ones have an unfair advantage since they made it in before Apple stopped allowing apps in that use the Accessibility API (all window managers).
2. That they allow true trial versions of paid apps in the app store, not the IAP way of doing it.

> because from what I've seen, this one is simply the best.
Idea: Be able to create a saved window dimensions and position, maybe with some kind of helper tool to create them, with options like "window width %", etc.

Custom window dimension shortcuts are in my paid app, [Hookshot](https://hookshot.app). It is a superset of Rectangle.

**Comment by zacharyfreiberg** [comment]
The icon looks good to me and I don't think anything needs further changing. Just giving a second opinion.
